gdscwm/project_dispense
# Locate Unique Patron

# Closes #6 

## Changes
The Power Automate Flow now locates the patron with the specified email, initializes variables, and returns those back to the server. The code now parses the patron and returns it with proper typings.

## Testing
To test:

1. Start the server
2. Open the Swagger UI
3. In the `FlowController/test` endpoint, input a valid email:
```JSON
{"email":"abcdef@wm.edu"}
```
4. Test this with a non-existent email as well and see if you get the right error message.
5. See console output to see if the data is right.

Success should look something like this:
<img width="645" alt="image" src="https://user-images.githubusercontent.com/65508654/224517467-eb29c18c-0dbd-4f21-b2d9-b816c09813ec.png">
